spec/models/manageiq/providers/amazon/cloud_manager/refresher_spec.rb

```diff
@@ -34,6 +34,7 @@
       assert_specific_vm_powered_on
       assert_specific_vm_powered_off
       assert_specific_vm_on_cloud_network
+      assert_specific_vm_on_cloud_network_1
```

Comment:
could that be something more meaningful than `_1` ?
Has it something to do with floating ips?